styles/navigation/nav.css

```diff
@@ -0,0 +1,42 @@
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --primary-color: white;
+  --secondary-color: rgb(15, 1, 1);
```

Comment:
Can declare variables for colors

Reply:
will refactor for every component, thanks